gcp-client-proxy/serverless/Dockerfile

```diff
@@ -38,8 +38,16 @@ RUN mkdir -p /usr/local/gcloud \
 # Adding the package path to local
 ENV PATH $PATH:/usr/local/gcloud/google-cloud-sdk/bin
 
+# Setting environment variables for GCP client proxy
+# JSON file containing an "email" key and a "cred_filename" key
```

Comment:
This should just be the credentials directly from GCP, not a JSON file with those keys (the comment in `run_proxy.sh` was incorrect after the refactors to the python functions).